api/GPUDevice.json

```
@@ -653,6 +653,46 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "deprecate_depth_bias_lines_points": {
+          "__compat": {
+            "description": "<code>depthBias</code>, <code>depthBiasSlopeScale</code>, and <code>depthBiasClamp</code> must be <code>0</code> for line and point topologies.",
```

How about we avoid the "deprecate" notion, and focus on the behavior of the spec change:

```suggestion
        "validates_depth_bias_for_line_and_point_topologies": {
          "__compat": {
            "description": "Validates that <code>depthBias</code>, <code>depthBiasSlopeScale</code>, and <code>depthBiasClamp</code> must be <code>0</code> for line and point topologies.",
```

---

Yeah, I like this

---

I've updated the other instance too, so the wording matches.